(GH-8628) Note PSCustomObject Count Behavior (#8639)
MicrosoftDocs/PowerShell-Docs: reference/5.1/Microsoft.PowerShell.Core/About/about_Arrays.md

## Arrays of zero or one

Beginning in Windows PowerShell 3.0, a collection of zero or one object has the
**Count** and **Length** properties. Also, you can index into an array of one
object. This feature helps you to avoid scripting errors that occur when a
command that expects a collection gets fewer than two items.

> > [!TIP]
> > In Windows PowerShell, objects created by casting a **Hashtable** to
> > `[pscustomobject]` are an exception and return `$null` for the **Count** and
> > **Length** properties, though they do support the array indexing
> > functionality.

Reviewer: ```suggestion
> [!TIP]
In Windows PowerShell, objects created by casting a **Hashtable** to
`[pscustomobject]` do not have the **Length** or **Count** properties.
Attempting to access these members returns `$null`.
```

> 

Author: I am unsure of the wording here but I do feel it is useful to call out the behavior here as well as in `about_PSCustomObject` as it directly contradicts the stated documentation.
